src/main/java/com/graphhopper/converter/ConverterConfiguration.java

```diff
@@ -127,11 +134,63 @@ public void setIPWhiteList(String ipWhiteList) {
         this.ipWhiteList = ipWhiteList;
     }
 
+    @JsonProperty
     public String getNominatimReverseUrl() {
         return nominatimReverseUrl;
     }
 
+    @JsonProperty
     public void setNominatimReverseUrl(String nominatimReverseUrl) {
         this.nominatimReverseUrl = nominatimReverseUrl;
     }
+
+    @JsonProperty
+	public String getGisgraphyGeocodingUrl() {
+		return gisgraphyGeocodingUrl;
+	}
+
+    @JsonProperty
+	public void setGisgraphyGeocodingUrl(String gisgraphyGeocodingUrl) {
```

Comment:
Here URL is Url, but in the converter.yml it is URL. Also the reverseGeocodingURL is all in upper case as well. Since the other services use Url, it might be the best to have everything as "Url"? We could also rename everything "URL", since URL [is the common acronym](https://en.wikipedia.org/wiki/URL).

Reply:
It's interesting that this wasn't picked up the tests. Maybe we need better integration tests? When I tried to start the server locally it wouldn't start due to that

Reply:
ok, I rename all to Upper.